### ISSUE TITLE
provide a raspberry-pi friendy version

### ISSUE THREAD

**Issue by CircleCode** [lesspass/lesspass#215]
Current docker-compose version won't run on rasberry pi because images are not arm.
It would be useful in order to easy self hosting to provide rpi images, or at least rpi docker compose file.

Ideally, this file would not only use arm images, but would also use images with a maximum of shared layers to limit their cost on the raspberry-pi

we need to choose base images for

- [ ] postgres
- [ ] backend (python)
- [ ] frontend (node)
- [ ] nginx

**Comment by CircleCode** [comment]
for postgres, [rotschopf/rpi-postgres](https://hub.docker.com/r/rotschopf/rpi-postgres/) seeems a good candidate

**Comment by CircleCode** [comment]
for backend, [hypriot/rpi-node/](https://hub.docker.com/r/hypriot/rpi-node/) seems a good candidate

**Comment by CircleCode** [comment]
for backend, I'd consider [hypriot/rpi-python/](https://hub.docker.com/r/hypriot/rpi-python/) but it is based on a third image (a raspbian variant)

**Comment by guillaumevincent** [comment]
After a discussion with you on IRC here is my conclusion:

  * I am against the idea of maintaining custom docker images for raspberry-pi. First it takes time. And we do not have a lot of time. Then I prefer a docker image reviewed and maintained by a large number of people.

  * It may be interesting to think about integrating sqlite for the hosted version. That would solve the database problem. This removes a container. It must be possible to do very easily with environment variables.

  * To finish docker is good, but one can also dig the side of snap on raspberry pi.

TLDR: We will not officially create docker image for raspberry pi, but we can facilitate self-hosting so that it works well on raspberry pi

**Comment by guillaumevincent** [comment]
I close this one
Docker images are multiplatform: https://blog.docker.com/2017/09/docker-official-images-now-multi-platform/

We can close this now, it should work on raspberry pi too
